transport/tchannel/channel_outbound.go

```diff
@@ -195,7 +195,9 @@ func (o *ChannelOutbound) Call(ctx context.Context, req *transport.Request) (*tr
 	}
 
 	err = getResponseError(headers)
-	deleteReservedHeaders(headers)
+	// no check: err will be returned as is
```

Comment:
is this comment intentional?

Reply:
Yeah, for some reason I was asking myself "where's error handling?" every time I look at this line, so I decided to left a comment